app/services/status_message_creation_service.rb

```diff
@@ -19,20 +19,9 @@ def create(params)
 
   def build_status_message(params)
     public = params[:public] || false
-    filter_mentions params
```

Comment:
fixed in https://github.com/diaspora/diaspora/pull/6818/commits/56dd62ceb39ee0f372eb549a71ac0df66a4f51c9